editor/README.md

```diff
@@ -0,0 +1,18 @@
+Editor: an overview
+===================
+
+The logic flow concerning the editor includes: inferring a block representation of the post content (parsing); describing the state of a post (representation); rendering of the post to the DOM (rendering); attaching controls to manipulate the content a.k.a blocks (UI).
+
+<img src="https://docs.google.com/drawings/d/1iuownt5etcih7rMMvPvh0Mny8zUA1Z28saxjxaWmfJ0/pub?w=1234&amp;h=453">
```

Comment:
Cool diagram. A few thoughts:

- Should we make a distinction about how the "saved post" is the source of truth as far as a persisted post goes, but the "post state" is the source of truth during the lifetime of the editor once it's been parsed?
- As you have `wp.blocks.parse( post.content.raw )` would it similarly be appropriate to annotate `wp.blocks.serialize( blocks )` at the serialization step?
- I like the singular data flow of visual editor manipulating blocks state, but the distinction between "Manipulation" and "Block" labels are a bit fuzzy to me. Are these on the level of a single block? The composed set of blocks of the editor?

Also, minor, but to avoid HTML in the Markdown, this could be written as:

```
![Diagram](https://docs.google.com/drawings/d/1iuownt5etcih7rMMvPvh0Mny8zUA1Z28saxjxaWmfJ0/pub?w=1234&h=453)
```